catalog/spatial/csw/spatial-csw-endpoint/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/endpoint/mappings/CswRecordMapperFilterVisitor.java

```diff
@@ -119,6 +108,40 @@ public CswRecordMapperFilterVisitor(MetacardType metacardType,
         }
     }
 
+    private static void convertGeometryExpressionToEpsg4326(Expression expression) {
+        if (expression instanceof LiteralExpressionImpl) {
+            LiteralExpressionImpl literalExpression = (LiteralExpressionImpl) expression;
+            Object valueObj = literalExpression.getValue();
+            if (valueObj instanceof Geometry) {
+                Geometry geometry = (Geometry) valueObj;
+                Object userDataObj = geometry.getUserData();
+                if (userDataObj instanceof CoordinateReferenceSystem) {
+                    CoordinateReferenceSystem sourceCRS = (CoordinateReferenceSystem) userDataObj;
+                    Geometry convertedGeometry = null;
+                    try {
+                        convertedGeometry = GeospatialUtil.transformToEPSG4326LonLatFormat(geometry,
+                                sourceCRS);
+                        literalExpression.setValue(convertedGeometry);
+                    } catch (GeoFormatException e) {
+                        LOGGER.trace("Unable to convert geometry to EPSG:4326 format", e);
```

Comment:
❓ Just trace? Is this just something that will really never happen and if it does an admin will want to explicitly turn up logging on this class to figure out why?